packages/components/pagination/src/components/pager.vue

```diff
@@ -51,9 +51,9 @@
         nsPager.is('disabled', disabled),
       ]"
       tabindex="0"
-      @mouseenter="onMouseenter('right')"
+      @mouseenter="onMouseenter()"
       @mouseleave="quickNextHover = false"
-      @focus="onFocus('right')"
+      @focus="onFocus()"
```

Comment:
```suggestion
      @mouseenter="onMouseenter"
      @mouseleave="quickNextHover = false"
      @focus="onFocus"
```

Reply:
We don't need to bracket here since no parameter were used

Reply:
In this case, a TypeScript type error occurs. 🤔

![image](https://user-images.githubusercontent.com/27342882/172985056-453cbc39-bbd1-4991-a31d-c46d917da48e.png)